### DISCUSSION TITLE
Pardon the interruption error

### DISCUSSION THREAD

**Discussion by EbaadAhmed** [cefsharp/CefSharp#4531] [General]
Hi i am facing this Pardon the interruption error when using Cefsharp windows v. 114, does anyone knows about this? 

![image](https://github.com/cefsharp/CefSharp/assets/16746751/265b879c-642a-446b-b71d-bb3162c2c424)

**Comment by amaitland** [comment]
Try https://github.com/cefsharp/CefSharp/discussions/4263#discussioncomment-3844085

**Reply by EbaadAhmed** [reply]
Thanks for the reply,
i have tried but it doesn't work, i have got the same result. It looks like some distill networks detects cefsharp as bot (although its not a bot but it thinks it is). It is happening on ticketmaster event page.

**Reply by EbaadAhmed** [reply]
Also i have tried `BrowserSettings` `Javascript`, `JavascriptAccessClipboard`, `JavascriptDomPaste` as `CefState.Enabled` but didn't work.

**Reply by amaitland** [reply]
Embedded browsers are often treated differently as they're easy to write bots with. From a `CefSharp` point of view there's nothing we can really do about that. 

Try changing the `UserAgent`. You can use `DevTools` to analyse the web page javascript to see if they are doing some client side detection,

If you have a legitimate use case for accessing the website then you may need to contact the website owner for a resolution.

**Reply by bwojdyna** [reply]
Are you fix this? any  CEFSharp setting?

**Reply by aimcosoftware** [reply]
There isn't anything to fix, the server is detecting a browser it doesn't like.
